lldb/source/Commands/CommandCompletions.cpp

```diff
@@ -297,8 +312,9 @@ class ModuleCompleter : public Completer {
   void DoCompletion(SearchFilter *filter) override { filter->Search(*this); }
 
 private:
-  const char *m_file_name;
-  const char *m_dir_name;
+  std::optional<llvm::StringRef> m_file_name;
```

Comment:
Unless there's another constructor I'm not seeing, m_file_name is always assigned to so there's always some value in the optional. Should it just be stringref?

Reply:
Yeah, this was my mistake. I originally wrote this using `"/"` as the invalid file name (thinking `/` cannot appear in a file name), but then I changed it because: a) I though it was too clever; b) I realized that if `"/"` is the whole path, then `/` *is* the file name.  And then I only updated half of the code...